spawn.c

```diff
@@ -452,18 +475,20 @@ child_exit_callback(GPid pid, gint status, gpointer user_data)
  * @tparam[opt=nil] function exit_callback Function to call on process exit. The
  * function arguments will be type of exit ("exit" or "signal") and the exit
  * code / the signal number causing process termination.
+ * @tparam[opt=nil] table cmd The environment to use for the spawned program.
+ * Without this, the spawned process inherits awesome's environment.
```

Comment:
I prefer to have two spaces of extra indentation for continued lines.
It's not the case for other params, but could be done for the new one.

I think there should be no comma after "Without this".